precompiles/staking/method_get_all_validators_test.go

```diff
@@ -0,0 +1,57 @@
+package staking
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/node/testutil/sample"
+)
+
+func Test_GetAllValidators(t *testing.T) {
+	t.Run("should return empty array if validators not set", func(t *testing.T) {
+		// ARRANGE
+		s := newTestSuite(t)
+
+		// Clean all validators.
+		validatorsList := s.sdkKeepers.StakingKeeper.GetAllValidators(s.ctx)
+		for _, v := range validatorsList {
+			s.sdkKeepers.StakingKeeper.RemoveValidator(s.ctx, v.GetOperator())
+		}
+
+		methodID := s.contractABI.Methods[GetAllValidatorsMethodName]
+		s.mockVMContract.Input = methodID.ID
+
+		// ACT
+		validators, err := s.contract.Run(s.mockEVM, s.mockVMContract, false)
+
+		// ASSERT
+		require.NoError(t, err)
+
+		res, err := methodID.Outputs.Unpack(validators)
+		require.NoError(t, err)
+
+		require.Empty(t, res[0])
+	})
+
+	t.Run("should return validators if set", func(t *testing.T) {
```

Comment:
The size of validatorRes might be quite big (100 + validators) . Can we add a unit test with a big set .

Reply:
As before, this method is out of scope.

Reply:
Can we add a note for this, in an issue? I think it would be an important thing to check

Reply:
Sure, created https://github.com/zeta-chain/node/issues/3062